CHARTER.md

```diff
@@ -1,59 +1,61 @@
+Proposals may only be made asychronously via the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) by submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). The provided [template file](art-decisions/proposals/template.md) may be used as a guideline for new proposals. Proposals may only be submitted by current ART members. If you would like to encourage discussion around a proposal before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not make any changes to your proposal.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a minimum of 4 votes is required with 75% approval majority among the voters.
+Each member of the ART is eligible for voting if they have been a member for at least 30 days. For any proposal that impacts the whole project or processes, **approval requires a minimum of four votes with 75% approval among those voting**.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 1 month after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Any vote must be open for at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal in calendar days. The voting period begins on the day the pull request is opened for review and ends at midnight [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) on the last day of the voting period. Proposals can be extended for no longer than 30 days total. Extensions can be made before the end of the current voting period by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension. **If approval is not reached within the voting period deadline, the proposal is rejected.**
 
-Any voting period must last at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal. Proposals can be extended no more than twice and for no longer than 30 days total. Extensions can be made by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension.
+A rejected proposal may only be submitted to the ART again after a miniumum of 30 days following a rejection. If the same proposal is rejected twice, it will not be considered again by the ART.
```

Comment:
Note that this requires "someone" to officially keep track of all rejected proposals and doing something like immediately closing any duplicate proposal that's been rejected twice. Not a big deal right now, but could be if it catches on. And how close does the concept and/or wording need to be to qualify as a duplicate?

Is this restriction really necessary or useful? So what if someone makes the same proposal again the next year? (Maybe the market conditions have changed and what seemed radical and impractical before is now quite sensible or even obvious? If someone said "apply AI" twice three years ago and it was rejected as impractical, does that mean we're now not allowed to consider applying AI?)

Reply:
I tend to agree. This is definitely some legacy language in the charter that I'm not sure is adding anything except maybe avoiding noise from someone ruthlessly trying to push a proposal through, submitting it again and again. Maybe we keep the 30-day minimum delay but remove the the twice-rejected limit?

As far as what constitutes a duplicate, that's a tough one, but I'm hesitant to get so pedantic in the charter right not to describe what a duplicate is. We are all adults... right?

Reply:
Dropping the twice-rejected limit would be good. We can say something about malicious (or "nuisance") repetition, but that's hard to define and would be harder to enforce.